src/System.Text.Json/tests/JsonNode.TraversalTests.cs

```diff
@@ -198,5 +219,61 @@ public static void TestParseWithDuplicates()
 
             Assert.Throws<ArgumentException>(() => JsonNode.Parse(stringWithDuplicates, default, DuplicatePropertyNameHandling.Error));
         }
+
+        [Theory]
+        [InlineData(DuplicatePropertyNameHandling.Replace)]
```

Comment:
We need to update this PR to react to this change: https://github.com/dotnet/corefx/pull/41030

`DuplicatePropertyNameHandling` -> `DuplicatePropertyNameHandlingStrategy`

And use `JsonNodeOptions`.

Reply:
To clarify, @kasiabulat, this is just a note as a reminder for either me or @joperezr to take this PR forward. I am not suggesting you react to this change/feedback :)

Reply:
Sure :)